discovery-provider/src/queries/queries.py

```diff
@@ -1461,7 +1462,12 @@ def get_users_account():
         else:
             return api_helpers.error_response('Invalid wallet length', 400)
 
-        user = base_query.one()
+        # If user cannot be found, exit early and return empty response
+        try:
+            user = base_query.one()
+        except NoResultFound:
+            return api_helpers.success_response([])
```

Comment:
Let's make this success_response(None)

since the output right now has data as just the user object 
https://discoveryprovider.audius.co/users/account?wallet=0x7d273271690538cf855e5b3002a0dd8c154bb060

[] doesn't make all that much sense.

Then, on the libs side, we can just check 

const userAccount = await this.getUserAccount(this.web3Manager.getWalletAddress())
if (userAccount) this.userStateManager.setCurrentUser(userAccount)

which is the existing check we have.

Reply:
if we want to maintain consistency with returning an empty array if a wallet is not found, it might be best to do that and have an additional if check like:
```
if (userAccount && userAccount.length > 0) this.userStateManager.setCurrentUser(userAccount)

Reply:
chatted offline, [] would be consistent w/ the other endpoint, but not this once since it never returns an array